packages/cli/src/lib/check.js

```diff
@@ -1,8 +1,42 @@
+const { listFilesMatchPattern } = require('./fast-glob.js')
+const { cosmiconfig } = require('cosmiconfig')
+const { logger } = require('./logger.js')
+
 async function check(args) {
-  const { pattern, traces } = args
+  const { pattern, configFilePath } = args
+  let files = await listFilesMatchPattern(pattern)
+  logger.debug('List of files that match pattern:\n' + files.join('\r\n'))
+
+  const explorer = cosmiconfig('bv')
+  if (configFilePath === undefined) {
+    logger.info(
+      'config option not specified, searching for default file path bv.config.js, .bvrc.json, etc.'
+    )
 
-  console.log('pattern=', pattern)
-  console.log('traces=', traces)
+    try {
+      const result = await explorer.search()
+      if (result == null) {
+        logger.error('Default config file not found')
+        process.exit(0)
```

Comment:
Please change it to `process.exit(1)` as 0 exit code means success, non-zero exit codes are for errors.